### ISSUE TITLE
Errors when importing external ES6 module as well as internal module (webpack resolver)

### ISSUE THREAD

**Issue by nhardy** [benmosher/eslint-plugin-import#661]
There seems to be a strange issue with `eslint-import-resolver-webpack` where when importing an internal module (webpack resolve alias) in conjunction with an external ES6 module (not transpiled).

```javascript
import { get } from 'lodash-es'; // external es6 module
import Thing from 'app/components/Thing'; // internal module
import anotherThing from './relative/path'; // relative path
```

When this occurs, there are a bunch of errors on the `lodash-es` import complaining: `Resolve error: Unexpected reserved word`.
The internal module also has an error complaining that `'app' should be listed in the project's dependencies`.

Removing either the external es6 module _or_ the internal module fixes _both_ of these errors, however.

**Comment by nhardy** [comment]
I think #666 is the same issue.

**Comment by cattermo** [comment]
I have the same problem. Any idea of how to solve this? It makes the webpack resolver unusable for me.

**Comment by soryy708** [comment]
Does https://github.com/import-js/eslint-plugin-import/issues/772 help?
